### PR TITLE
SAK-49732 push. Use bigger varchar for the PUSH_SUBSCRIPTIONS column

### PR DESCRIPTION
**Pull request by adrianfish**
https://sakaiproject.atlassian.net/browse/SAK-49732

### CODE REVIEW

File: kernel/api/src/main/java/org/sakaiproject/messaging/api/model/PushSubscription.java [hunk 1]
```diff
@@ -60,7 +61,8 @@ public class PushSubscription implements PersistableEntity<Long> {
     @Column(name = "AUTH", length = 255, nullable = false)
     private String auth;
 
-    @Column(name = "ENDPOINT", length = 255, nullable = false)
+    @Lob
+    @Column(name = "ENDPOINT", nullable = false)
```
Comment (ottenhoff) [inline kernel/api/src/main/java/org/sakaiproject/messaging/api/model/PushSubscription.java:65]: what if we just change length to 2048?

Comment (adrianfish) [reply kernel/api/src/main/java/org/sakaiproject/messaging/api/model/PushSubscription.java:65]: I think you get the same thing, a longtext. I'll try it now.

Comment (adrianfish) [reply kernel/api/src/main/java/org/sakaiproject/messaging/api/model/PushSubscription.java:65]: No, you're right. You get a varchar(2048). That'll do.

Comment (adrianfish) [reply kernel/api/src/main/java/org/sakaiproject/messaging/api/model/PushSubscription.java]: I think I'm still stuck on mysql 5.0.2 ...

Comment (ottenhoff) [reply kernel/api/src/main/java/org/sakaiproject/messaging/api/model/PushSubscription.java]: v true maxlength=255 is embedded in my brain for varchar

**Review by ottenhoff — APPROVED** [review]

**Pull request merged**